bin/civihr-install

```diff
@@ -0,0 +1,820 @@
+#!/bin/bash
```

Comment:
So if we add a extension don't you think it will cause an extra step to end user to rename file in-case of user doesn't want to use curl command to rename and directly save the file to use it. And also executing civihr-install.sh <parameters> would be not so user friendly.